stackstorm/Dockerfile

```diff
@@ -5,8 +5,8 @@ LABEL com.stackstorm.version="${ST2_VERSION}"
 
 RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev
 
-ADD ./st2bundle*.deb /tmp/
-RUN dpkg -i /tmp/st2bundle*.deb && rm -r /tmp/* && apt-get clean
+ADD ./st2*.deb /tmp/
+RUN dpkg -i /tmp/st2*.deb && rm -r /tmp/* && apt-get clean
```

Comment:
this is not precisely correct... if you have anything else there like st2auth, it will be installed, it's not that we want I guess.